keywords/MobileRestClient.py

```diff
@@ -1549,15 +1543,32 @@ def verify_docs_present(self, url, db, expected_docs, auth=None, timeout=CLIENT_
                     missing_docs.append(resp_doc)
                     all_docs_returned = False
 
+                if attachments:
+                    # Check for an attachment
+                    doc_id = resp_doc["key"]
+                    doc_data = self._session.get("{}/{}/{}".format(url, db, doc_id))
```

Comment:
We did not. Made some changes to include attachment data and explicitly verify attachments